externs/olx.js

```diff
@@ -1110,6 +1111,12 @@ olx.control.FullScreenOptions.prototype.keys;
  */
 olx.control.FullScreenOptions.prototype.target;
 
+/**
+ * Target Element for FullScreen.
+ * @type {Element|string|undefined}
+ * @api
+ */
+olx.control.FullScreenOptions.prototype.fullScreenTarget;
```

Comment:
We usually use "target" for the destination.  For the control itself, the "target" is where the "controlling element" gets rendered (for example, a button to activate the control).

In this case, you want to provide an option for overriding the source element (not the destination).  I'd be tempted to call this option `source`.  In addition, it needs better documentation.  You should say that by default, the map is the source.  If the user would like a different element to be rendered in full-screen mode, they can provide a custom source.